### ISSUE TITLE
spanish version ?

### ISSUE THREAD

**Issue by vlasvlasvlas** [osmlab/teachosm#76]
its ok if i translate this into spanish version /es?

thanks

**Comment by geomantic** [comment]
Hi Vladimir,
Yes, of course! Your initiative to translate is more than welcome. As a guide, I'd look to LearnOSM for how they are managing translations and localizations. And if you need help with process, this list is a good place to voice concerns.
Thanks much for taking this on, and best regards,

--SEJ

**Comment by mikelmaron** [comment]
Fantastic. Vladimar, are you comfortable working with Jekyll to build in the "es" translated section? * Mikel Maron * +14152835207 @mikel s:mikelmaron 

     On Wednesday, February 18, 2015 8:18 PM, Steven Johnson <notifications@github.com> wrote:
   
 

 Hi Vladimir,
Yes, of course! Your initiative to translate is more than welcome. As a guide, I'd look to LearnOSM for how they are managing translations and localizations. And if you need help with process, this list is a good place to voice concerns.
Thanks much for taking this on, and best regards,

--SEJ

**Comment by mikel** [comment]
Hi there Mikel Maron @mikelmaron , every time you post into a thread on github, you @ mention my github handle... any chance you could remove the @mikel from your emails when you post to github?  I get several a week :)

**Comment by mikelmaron** [comment]
@mikel I had no idea! will do

**Comment by mikel** [comment]
Thanks @mikelmaron  :)

**Comment by geomantic** [comment]
@vlasvlasvlas Hi Vladimiro, just going through the issues on TeachOSM and noticed this outstanding issue. Is it still your intent to translate these materials into Spanish?

**Comment by vlasvlasvlas** [comment]
yep i will! i just make a /es/ folder and start translatin /en/?

**Comment by geomantic** [comment]
@vlasvlasvlas Yes, make an /es/ folder. What's the best way to do that? Fork the gh-pages branch?

**Comment by Nick-Tallguy** [comment]
This link may help with info about adding languages - at some point I hope to get the time to add this to the main LearnOSM site as well - https://github.com/Nick-Tallguy/Nick-Tallguy.github.io/wiki/Languages-DRAFT

**Comment by Nick-Tallguy** [comment]
Added files ready for translation. When you are ready, the following files will also need amending;  
- all-translation.json    
- _config.yml    
- /_layouts/default.html
